_posts/2021-02-23-welcome-to-my-blog.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Welcome to my personal blog!"
+date:   2021-02-23 23:46:41 +0100
+categories: blog
+---
+
+## Blog reactivation
+
+I created my first blog in 2005. I published several posts, but I was a very shame of what I wrote. Nevertheless, from my readers, I was able to gain new friendships that widen my perspective on the World so, in retrospect, it was all very good for me.
+
+Therefore, I thought that I will reactivate this blog and 2010, then in 2015, then in 2019, etc. but all those attempts failed.
+
+Now I have not only motivation, but I also have a plan for that.
+
+I will reserve time in my calendar strictly for writing and I will publish weekly some of my thoughts.
+
+The most important part is that I will have a Mentor/Guru/Guide that will help me to deliver this plan. Ash Furrow is a world-class software developer that I highly admire. I humbly asked him for his time&knowleadge to help me grow as a person, team leader, and software developer. He agreed only under one condition. That I will publish my efforts publicly.
+
+I directly said yes, because I truly believe that it could help others too. I enjoy sharing my knowledge and experience with others. I like providing trainings and teaching others, however, I almost always feel very unprepared and I always have self-doubts (even Ash mentioned that [his blog post](https://ashfurrow.com/blog/normalizing-struggle/)).
+
+Only closest to me, people know that I am a very big introvert. I just prefer to be alone and to try to learn and solve everything by myself. I was already quite successful in plenty of different topics by doing this so far. I also know that I am in a very good situation because my whole life I was working very hard, very often even up to 16 hours a day to say that:
```

Comment:
Small grammatical error:

```suggestion
Only the people closest to me know that I am a very big introvert. I just prefer to be alone and to try to learn and solve everything by myself. I was already quite successful in plenty of different topics by doing this so far. I also know that I am in a very good situation because my whole life I was working very hard, very often even up to 16 hours a day to say that:
```

Reply:
I also feel like an introvert that everything thinks is an extrovert. [I wrote a blog post about it](https://ashfurrow.com/blog/ambiversion/) a while ago. Sometimes I am one, and sometimes I am the other.